### DISCUSSION TITLE
LocalImageService

### DISCUSSION THREAD

**Discussion by gp-slick-coder** [reupen/photos.reupen.uk#31] [General]
Your LocalImageService with the 'Picture' control looks interesting. I will try that out. Thanks for sharing.

**Comment by reupen** [comment]
No worries. Of course Astro has similar functionality out of the box. 😅

The custom image service was mainly because I wanted to retain EXIF data, embed a colour profile in WebP images, and also customise some conversion options (e.g. effort).

Astro v3 didn't have a Picture component when I wrote my one, but the one that's since been added to Astro probably doesn't do exactly what I want either.
